Controls/Cafe/Customer1.c

```diff
@@ -11,6 +13,22 @@
  */
 void Customer1_checkout(char** name, int* cost) {
 	OS_ERR err; // Make sure to check for errors and print the error code if not OS_ERR_NONE
+	err = OS_ERR_NONE;
```

Comment:
While in this case, you do have to set err to OS_ERR_NONE, typically we'll just declare an OS_ERR err without setting it to anything since it will get set during function calls.

This is really an issue with the starter project since it says to check for errors, but there's actually nothing that sets err and requires error-checking in this function (whoops)